kornia/geometry/transform/imgwarp.py

```diff
@@ -173,9 +189,38 @@ def warp_affine(
 
     grid = F.affine_grid(src_norm_trans_dst_norm[:, :2, :], [B, C, dsize[0], dsize[1]], align_corners=align_corners)
 
+    if padding_mode == "fill":
+        return _warp_and_fill(src, grid, align_corners=align_corners, mode=mode, fill_value=fill_value)
     return F.grid_sample(src, grid, align_corners=align_corners, mode=mode, padding_mode=padding_mode)
 
 
+def _warp_and_fill(
```

Comment:
```suggestion
def _fill_and_warp(
```
thinking it well - actually better name since that's the order of execution